browser/src/canvas/CanvasSectionContainer.ts

```diff
@@ -584,6 +584,7 @@ class CanvasSectionContainer {
 			double-check the document container size to see if we've missed a resize event
 		*/
 		if (this.width === 0 || this.height === 0) {
+			console.warn('Canvas width or height is zero.');
```

Comment:
we should use our logger: `app.console.warn`
then it is included in the "show/hide" protocol logs feature